### DISCUSSION TITLE
Debian 11 install

### DISCUSSION THREAD

**Discussion by knightflyer** [ltsp/ltsp#624] [General]
I am abandoning Mint due to NFS doesn't seem to work with 20.2.  Even with stanadlone pcs NFS is squirrely.  I revert back to mint pre-19 and I have no issues with NFS.   The server I want to replace is mint 17.3 and is NFS only. "No ltsp".  Since I cannot get NFS to work with Mint 20.2 then it makes since ltsp is not the issue with Mint 20.2, it is NFS.

After deciding to migrate to Debian current problem is Debian 11 install
I have tried installing by adding the ppa using the debian wget on the ltsp.org. 
that ends up with multiple Possible missing firmware for module i915
I have tried using just the ltsp that is in the debian repository, it installs 21.01-01 but cannot execute ltsp

What am I doing wrong?

**Comment by alkisg** [comment]
Hi, paste the whole terminal output so that we see the commands you're running and the exact output you're getting.

The "missing firmware" messages aren't related to LTSP.

**Reply by knightflyer** [reply]
The output in question is at the end. (sorry)

root@littledebi:/home/ee# wget https://ltsp.org/misc/ltsp-ubuntu-ppa-focal.list -O /etc/apt/sources.list.d/ltsp-ubuntu-ppa-focal.list
wget https://ltsp.org/misc/ltsp_ubuntu_ppa.gpg -O /etc/apt/trusted.gpg.d/ltsp_ubuntu_ppa.gpg
--2021-12-03 11:00:56--  https://ltsp.org/misc/ltsp-ubuntu-ppa-focal.list
Resolving ltsp.org (ltsp.org)... 185.199.108.153, 185.199.110.153, 185.199.111.153, ...
Connecting to ltsp.org (ltsp.org)|185.199.108.153|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 118 [text/plain]
Saving to: ‘/etc/apt/sources.list.d/ltsp-ubuntu-ppa-focal.list’

/etc/apt/sources.li 100%[===================>]     118  --.-KB/s    in 0s      

2021-12-03 11:00:57 (2.71 MB/s) - ‘/etc/apt/sources.list.d/ltsp-ubuntu-ppa-focal.list’ saved [118/118]

--2021-12-03 11:00:57--  https://ltsp.org/misc/ltsp_ubuntu_ppa.gpg
Resolving ltsp.org (ltsp.org)... 185.199.108.153, 185.199.110.153, 185.199.111.153, ...
Connecting to ltsp.org (ltsp.org)|185.199.108.153|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1123 (1.1K) [application/octet-stream]
Saving to: ‘/etc/apt/trusted.gpg.d/ltsp_ubuntu_ppa.gpg’

/etc/apt/trusted.gp 100%[===================>]   1.10K  --.-KB/s    in 0s      

2021-12-03 11:00:57 (25.6 MB/s) - ‘/etc/apt/trusted.gpg.d/ltsp_ubuntu_ppa.gpg’ saved [1123/1123]

root@littledebi:/home/ee# 
root@littledebi:/home/ee# 
root@littledebi:/home/ee# apt update
Hit:1 http://security.debian.org/debian-security bullseye-security InRelease
Hit:2 http://deb.debian.org/debian bullseye InRelease
Hit:3 http://deb.debian.org/debian bullseye-updates InRelease
Get:4 http://ppa.launchpad.net/ltsp/ppa/ubuntu focal InRelease [18.0 kB]
Get:5 http://ppa.launchpad.net/ltsp/ppa/ubuntu focal/main amd64 Packages [1,340 B]
Get:6 http://ppa.launchpad.net/ltsp/ppa/ubuntu focal/main Translation-en [960 B]
Fetched 20.3 kB in 2s (9,428 B/s)                             
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
root@littledebi:/home/ee# apt install --install-recommends ltsp ltsp-binaries dnsmasq nfs-kernel-server openssh-server squashfs-tools ethtool net-tools epoptes
gpasswd -a administrator epoptes
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
openssh-server is already the newest version (1:8.4p1-5).
openssh-server set to manually installed.
The following additional packages will be installed:
  epoptes-client faketime iperf keyutils libfaketime libfltk-images1.3
  libfltk1.3 libnfsidmap2 libtcl8.6 libtk8.6 libutempter0 libvncserver1
  nfs-common python3-attr python3-automat python3-click python3-colorama
  python3-constantly python3-hamcrest python3-hyperlink python3-incremental
  python3-netifaces python3-openssl python3-pyasn1 python3-pyasn1-modules
  python3-service-identity python3-twisted python3-twisted-bin
  python3-zope.interface rpcbind screen socat sshfs tcl tcl8.6 tigervnc-viewer
  tk tk8.6 x11vnc xterm
Suggested packages:
  resolvconf open-iscsi watchdog python-attr-doc python-openssl-doc
  python3-openssl-dbg python3-tk python3-qt4 python3-serial python3-wxgtk2.8
  python3-twisted-bin-dbg byobu | screenie | iselect tcl-tclreadline
  tigervnc-common xfonts-cyrillic
The following NEW packages will be installed:
  dnsmasq epoptes epoptes-client ethtool faketime iperf keyutils libfaketime
  libfltk-images1.3 libfltk1.3 libnfsidmap2 libtcl8.6 libtk8.6 libutempter0
  libvncserver1 ltsp ltsp-binaries net-tools nfs-common nfs-kernel-server
  python3-attr python3-automat python3-click python3-colorama
  python3-constantly python3-hamcrest python3-hyperlink python3-incremental
  python3-netifaces python3-openssl python3-pyasn1 python3-pyasn1-modules
  python3-service-identity python3-twisted python3-twisted-bin
  python3-zope.interface rpcbind screen socat squashfs-tools sshfs tcl tcl8.6
  tigervnc-viewer tk tk8.6 x11vnc xterm
0 upgraded, 48 newly installed, 0 to remove and 0 not upgraded.
Need to get 10.7 MB of archives.
After this operation, 41.4 MB of additional disk space will be used.
Do you want to continue? [Y/n] 
Get:1 http://deb.debian.org/debian bullseye/main amd64 rpcbind amd64 1.2.5-9 [51.4 kB]
Get:2 http://security.debian.org/debian-security bullseye-security/main amd64 squashfs-tools amd64 1:4.4-2+deb11u2 [135 kB]
Get:3 http://deb.debian.org/debian bullseye/main amd64 dnsmasq all 2.85-1 [32.0 kB]
Get:4 http://deb.debian.org/debian bullseye/main amd64 libfaketime amd64 0.9.8-9 [40.1 kB]
Get:5 http://deb.debian.org/debian bullseye/main amd64 faketime amd64 0.9.8-9 [15.9 kB]
Get:6 http://deb.debian.org/debian bullseye/main amd64 iperf amd64 2.0.14a+dfsg1-1 [102 kB]
Get:7 http://deb.debian.org/debian bullseye/main amd64 python3-netifaces amd64 0.10.9-0.2+b3 [17.3 kB]
Get:8 http://deb.debian.org/debian bullseye/main amd64 python3-openssl all 20.0.1-1 [53.7 kB]
Get:9 http://deb.debian.org/debian bullseye/main amd64 python3-attr all 20.3.0-1 [52.9 kB]
Get:10 http://deb.debian.org/debian bullseye/main amd64 python3-automat all 20.2.0-1 [27.5 kB]
Get:11 http://deb.debian.org/debian bullseye/main amd64 python3-constantly all 15.1.0-2 [8,356 B]
Get:12 http://deb.debian.org/debian bullseye/main amd64 python3-hyperlink all 19.0.0-2 [36.0 kB]
Get:13 http://deb.debian.org/debian bullseye/main amd64 python3-incremental all 17.5.0-1 [13.5 kB]
Get:14 http://deb.debian.org/debian bullseye/main amd64 python3-zope.interface amd64 5.2.0-1 [142 kB]
Get:15 http://deb.debian.org/debian bullseye/main amd64 python3-twisted-bin amd64 20.3.0-7 [22.5 kB]
Get:16 http://deb.debian.org/debian bullseye/main amd64 python3-pyasn1 all 0.4.8-1 [63.8 kB]
Get:17 http://deb.debian.org/debian bullseye/main amd64 python3-pyasn1-modules all 0.2.1-1 [34.6 kB]
Get:18 http://deb.debian.org/debian bullseye/main amd64 python3-service-identity all 18.1.0-6 [12.6 kB]
Get:19 http://deb.debian.org/debian bullseye/main amd64 python3-hamcrest all 1.9.0-3 [27.6 kB]
Get:20 http://deb.debian.org/debian bullseye/main amd64 python3-twisted all 20.3.0-7 [1,962 kB]
Get:21 http://deb.debian.org/debian bullseye/main amd64 socat amd64 1.7.4.1-3 [370 kB]
Get:22 http://deb.debian.org/debian bullseye/main amd64 libtcl8.6 amd64 8.6.11+dfsg-1 [1,018 kB]
Get:23 http://ppa.launchpad.net/ltsp/ppa/ubuntu focal/main amd64 ltsp all 21.01-1+202111030942~ubuntu20.04.1 [78.1 kB]
Get:24 http://deb.debian.org/debian bullseye/main amd64 libtk8.6 amd64 8.6.11-2 [780 kB]
Get:25 http://deb.debian.org/debian bullseye/main amd64 tk8.6 amd64 8.6.11-2 [72.3 kB]
Get:26 http://deb.debian.org/debian bullseye/main amd64 tcl8.6 amd64 8.6.11+dfsg-1 [124 kB]
Get:27 http://deb.debian.org/debian bullseye/main amd64 tcl amd64 8.6.11+1 [5,788 B]
Get:28 http://deb.debian.org/debian bullseye/main amd64 tk amd64 8.6.11+1 [5,828 B]
Get:29 http://deb.debian.org/debian bullseye/main amd64 libvncserver1 amd64 0.9.13+dfsg-2 [233 kB]
Get:30 http://deb.debian.org/debian bullseye/main amd64 x11vnc amd64 0.9.16-7 [1,017 kB]
Get:31 http://deb.debian.org/debian bullseye/main amd64 libutempter0 amd64 1.2.1-2 [8,960 B]
Get:32 http://deb.debian.org/debian bullseye/main amd64 xterm amd64 366-1 [819 kB]
Get:33 http://deb.debian.org/debian bullseye/main amd64 libfltk1.3 amd64 1.3.5-3 [566 kB]
Get:34 http://deb.debian.org/debian bullseye/main amd64 libfltk-images1.3 amd64 1.3.5-3 [62.9 kB]
Get:35 http://deb.debian.org/debian bullseye/main amd64 tigervnc-viewer amd64 1.11.0+dfsg-2 [271 kB]
Get:36 http://deb.debian.org/debian bullseye/main amd64 epoptes all 21.02-1 [184 kB]
Get:37 http://deb.debian.org/debian bullseye/main amd64 screen amd64 4.8.0-6 [609 kB]
Get:38 http://deb.debian.org/debian bullseye/main amd64 epoptes-client all 21.02-1 [24.9 kB]
Get:39 http://deb.debian.org/debian bullseye/main amd64 ethtool amd64 1:5.9-1 [182 kB]
Get:40 http://deb.debian.org/debian bullseye/main amd64 keyutils amd64 1.6.1-2 [52.8 kB]
Get:41 http://deb.debian.org/debian bullseye/main amd64 libnfsidmap2 amd64 0.25-6 [32.6 kB]
Get:42 http://deb.debian.org/debian bullseye/main amd64 net-tools amd64 1.60+git20181103.0eebece-1 [250 kB]
Get:43 http://deb.debian.org/debian bullseye/main amd64 nfs-common amd64 1:1.3.4-6 [232 kB]
Get:44 http://deb.debian.org/debian bullseye/main amd64 nfs-kernel-server amd64 1:1.3.4-6 [125 kB]
Get:45 http://deb.debian.org/debian bullseye/main amd64 python3-colorama all 0.4.4-1 [28.5 kB]
Get:46 http://deb.debian.org/debian bullseye/main amd64 python3-click all 7.1.2-1 [75.7 kB]
Get:47 http://deb.debian.org/debian bullseye/main amd64 sshfs amd64 3.7.1+repack-2 [46.0 kB]
Get:48 http://ppa.launchpad.net/ltsp/ppa/ubuntu focal/main amd64 ltsp-binaries all 21.01-1~ubuntu20.04.1 [551 kB]
Fetched 10.7 MB in 2s (4,872 kB/s)                  
Extracting templates from packages: 100%
Selecting previously unselected package rpcbind.
(Reading database ... 171512 files and directories currently installed.)
Preparing to unpack .../00-rpcbind_1.2.5-9_amd64.deb ...
Unpacking rpcbind (1.2.5-9) ...
Selecting previously unselected package dnsmasq.
Preparing to unpack .../01-dnsmasq_2.85-1_all.deb ...
Unpacking dnsmasq (2.85-1) ...
Selecting previously unselected package libfaketime:amd64.
Preparing to unpack .../02-libfaketime_0.9.8-9_amd64.deb ...
Unpacking libfaketime:amd64 (0.9.8-9) ...
Selecting previously unselected package faketime.
Preparing to unpack .../03-faketime_0.9.8-9_amd64.deb ...
Unpacking faketime (0.9.8-9) ...
Selecting previously unselected package iperf.
Preparing to unpack .../04-iperf_2.0.14a+dfsg1-1_amd64.deb ...
Unpacking iperf (2.0.14a+dfsg1-1) ...
Selecting previously unselected package python3-netifaces.
Preparing to unpack .../05-python3-netifaces_0.10.9-0.2+b3_amd64.deb ...
Unpacking python3-netifaces (0.10.9-0.2+b3) ...
Selecting previously unselected package python3-openssl.
Preparing to unpack .../06-python3-openssl_20.0.1-1_all.deb ...
Unpacking python3-openssl (20.0.1-1) ...
Selecting previously unselected package python3-attr.
Preparing to unpack .../07-python3-attr_20.3.0-1_all.deb ...
Unpacking python3-attr (20.3.0-1) ...
Selecting previously unselected package python3-automat.
Preparing to unpack .../08-python3-automat_20.2.0-1_all.deb ...
Unpacking python3-automat (20.2.0-1) ...
Selecting previously unselected package python3-constantly.
Preparing to unpack .../09-python3-constantly_15.1.0-2_all.deb ...
Unpacking python3-constantly (15.1.0-2) ...
Selecting previously unselected package python3-hyperlink.
Preparing to unpack .../10-python3-hyperlink_19.0.0-2_all.deb ...
Unpacking python3-hyperlink (19.0.0-2) ...
Selecting previously unselected package python3-incremental.
Preparing to unpack .../11-python3-incremental_17.5.0-1_all.deb ...
Unpacking python3-incremental (17.5.0-1) ...
Selecting previously unselected package python3-zope.interface.
Preparing to unpack .../12-python3-zope.interface_5.2.0-1_amd64.deb ...
Unpacking python3-zope.interface (5.2.0-1) ...
Selecting previously unselected package python3-twisted-bin:amd64.
Preparing to unpack .../13-python3-twisted-bin_20.3.0-7_amd64.deb ...
Unpacking python3-twisted-bin:amd64 (20.3.0-7) ...
Selecting previously unselected package python3-pyasn1.
Preparing to unpack .../14-python3-pyasn1_0.4.8-1_all.deb ...
Unpacking python3-pyasn1 (0.4.8-1) ...
Selecting previously unselected package python3-pyasn1-modules.
Preparing to unpack .../15-python3-pyasn1-modules_0.2.1-1_all.deb ...
Unpacking python3-pyasn1-modules (0.2.1-1) ...
Selecting previously unselected package python3-service-identity.
Preparing to unpack .../16-python3-service-identity_18.1.0-6_all.deb ...
Unpacking python3-service-identity (18.1.0-6) ...
Selecting previously unselected package python3-hamcrest.
Preparing to unpack .../17-python3-hamcrest_1.9.0-3_all.deb ...
Unpacking python3-hamcrest (1.9.0-3) ...
Selecting previously unselected package python3-twisted.
Preparing to unpack .../18-python3-twisted_20.3.0-7_all.deb ...
Unpacking python3-twisted (20.3.0-7) ...
Selecting previously unselected package socat.
Preparing to unpack .../19-socat_1.7.4.1-3_amd64.deb ...
Unpacking socat (1.7.4.1-3) ...
Selecting previously unselected package libtcl8.6:amd64.
Preparing to unpack .../20-libtcl8.6_8.6.11+dfsg-1_amd64.deb ...
Unpacking libtcl8.6:amd64 (8.6.11+dfsg-1) ...
Selecting previously unselected package libtk8.6:amd64.
Preparing to unpack .../21-libtk8.6_8.6.11-2_amd64.deb ...
Unpacking libtk8.6:amd64 (8.6.11-2) ...
Selecting previously unselected package tk8.6.
Preparing to unpack .../22-tk8.6_8.6.11-2_amd64.deb ...
Unpacking tk8.6 (8.6.11-2) ...
Selecting previously unselected package tcl8.6.
Preparing to unpack .../23-tcl8.6_8.6.11+dfsg-1_amd64.deb ...
Unpacking tcl8.6 (8.6.11+dfsg-1) ...
Selecting previously unselected package tcl.
Preparing to unpack .../24-tcl_8.6.11+1_amd64.deb ...
Unpacking tcl (8.6.11+1) ...
Selecting previously unselected package tk.
Preparing to unpack .../25-tk_8.6.11+1_amd64.deb ...
Unpacking tk (8.6.11+1) ...
Selecting previously unselected package libvncserver1:amd64.
Preparing to unpack .../26-libvncserver1_0.9.13+dfsg-2_amd64.deb ...
Unpacking libvncserver1:amd64 (0.9.13+dfsg-2) ...
Selecting previously unselected package x11vnc.
Preparing to unpack .../27-x11vnc_0.9.16-7_amd64.deb ...
Unpacking x11vnc (0.9.16-7) ...
Selecting previously unselected package libutempter0:amd64.
Preparing to unpack .../28-libutempter0_1.2.1-2_amd64.deb ...
Unpacking libutempter0:amd64 (1.2.1-2) ...
Selecting previously unselected package xterm.
Preparing to unpack .../29-xterm_366-1_amd64.deb ...
Unpacking xterm (366-1) ...
Selecting previously unselected package libfltk1.3:amd64.
Preparing to unpack .../30-libfltk1.3_1.3.5-3_amd64.deb ...
Unpacking libfltk1.3:amd64 (1.3.5-3) ...
Selecting previously unselected package libfltk-images1.3:amd64.
Preparing to unpack .../31-libfltk-images1.3_1.3.5-3_amd64.deb ...
Unpacking libfltk-images1.3:amd64 (1.3.5-3) ...
Selecting previously unselected package tigervnc-viewer.
Preparing to unpack .../32-tigervnc-viewer_1.11.0+dfsg-2_amd64.deb ...
Unpacking tigervnc-viewer (1.11.0+dfsg-2) ...
Selecting previously unselected package epoptes.
Preparing to unpack .../33-epoptes_21.02-1_all.deb ...
Unpacking epoptes (21.02-1) ...
Selecting previously unselected package screen.
Preparing to unpack .../34-screen_4.8.0-6_amd64.deb ...
Unpacking screen (4.8.0-6) ...
Selecting previously unselected package epoptes-client.
Preparing to unpack .../35-epoptes-client_21.02-1_all.deb ...
Unpacking epoptes-client (21.02-1) ...
Selecting previously unselected package ethtool.
Preparing to unpack .../36-ethtool_1%3a5.9-1_amd64.deb ...
Unpacking ethtool (1:5.9-1) ...
Selecting previously unselected package keyutils.
Preparing to unpack .../37-keyutils_1.6.1-2_amd64.deb ...
Unpacking keyutils (1.6.1-2) ...
Selecting previously unselected package libnfsidmap2:amd64.
Preparing to unpack .../38-libnfsidmap2_0.25-6_amd64.deb ...
Unpacking libnfsidmap2:amd64 (0.25-6) ...
Selecting previously unselected package ltsp.
Preparing to unpack .../39-ltsp_21.01-1+202111030942~ubuntu20.04.1_all.deb ...
Unpacking ltsp (21.01-1+202111030942~ubuntu20.04.1) ...
Selecting previously unselected package ltsp-binaries.
Preparing to unpack .../40-ltsp-binaries_21.01-1~ubuntu20.04.1_all.deb ...
Unpacking ltsp-binaries (21.01-1~ubuntu20.04.1) ...
Selecting previously unselected package net-tools.
Preparing to unpack .../41-net-tools_1.60+git20181103.0eebece-1_amd64.deb ...
Unpacking net-tools (1.60+git20181103.0eebece-1) ...
Selecting previously unselected package nfs-common.
Preparing to unpack .../42-nfs-common_1%3a1.3.4-6_amd64.deb ...
Unpacking nfs-common (1:1.3.4-6) ...
Selecting previously unselected package nfs-kernel-server.
Preparing to unpack .../43-nfs-kernel-server_1%3a1.3.4-6_amd64.deb ...
Unpacking nfs-kernel-server (1:1.3.4-6) ...
Selecting previously unselected package python3-colorama.
Preparing to unpack .../44-python3-colorama_0.4.4-1_all.deb ...
Unpacking python3-colorama (0.4.4-1) ...
Selecting previously unselected package python3-click.
Preparing to unpack .../45-python3-click_7.1.2-1_all.deb ...
Unpacking python3-click (7.1.2-1) ...
Selecting previously unselected package squashfs-tools.
Preparing to unpack .../46-squashfs-tools_1%3a4.4-2+deb11u2_amd64.deb ...
Unpacking squashfs-tools (1:4.4-2+deb11u2) ...
Selecting previously unselected package sshfs.
Preparing to unpack .../47-sshfs_3.7.1+repack-2_amd64.deb ...
Unpacking sshfs (3.7.1+repack-2) ...
Setting up python3-attr (20.3.0-1) ...
Setting up ltsp (21.01-1+202111030942~ubuntu20.04.1) ...
Created symlink /etc/systemd/system/multi-user.target.wants/ltsp.service → /lib/
systemd/system/ltsp.service.
Setting up net-tools (1.60+git20181103.0eebece-1) ...
Setting up squashfs-tools (1:4.4-2+deb11u2) ...
Setting up python3-colorama (0.4.4-1) ...
Setting up python3-zope.interface (5.2.0-1) ...
Setting up python3-openssl (20.0.1-1) ...
Setting up libfltk1.3:amd64 (1.3.5-3) ...
Setting up python3-automat (20.2.0-1) ...
Setting up python3-twisted-bin:amd64 (20.3.0-7) ...
Setting up sshfs (3.7.1+repack-2) ...
Setting up rpcbind (1.2.5-9) ...
Created symlink /etc/systemd/system/multi-user.target.wants/rpcbind.service → /l
ib/systemd/system/rpcbind.service.
Created symlink /etc/systemd/system/sockets.target.wants/rpcbind.socket → /lib/s
ystemd/system/rpcbind.socket.
Setting up python3-hamcrest (1.9.0-3) ...
Setting up python3-click (7.1.2-1) ...
Setting up libfltk-images1.3:amd64 (1.3.5-3) ...
Setting up libvncserver1:amd64 (0.9.13+dfsg-2) ...
Setting up dnsmasq (2.85-1) ...
Created symlink /etc/systemd/system/multi-user.target.wants/dnsmasq.service → /l
ib/systemd/system/dnsmasq.service.
Setting up ltsp-binaries (21.01-1~ubuntu20.04.1) ...
Setting up iperf (2.0.14a+dfsg1-1) ...
Setting up socat (1.7.4.1-3) ...
Setting up python3-incremental (17.5.0-1) ...
Setting up python3-hyperlink (19.0.0-2) ...
Setting up libtcl8.6:amd64 (8.6.11+dfsg-1) ...
Setting up tigervnc-viewer (1.11.0+dfsg-2) ...
update-alternatives: using /usr/bin/xtigervncviewer to provide /usr/bin/vncviewe
r (vncviewer) in auto mode
Setting up python3-netifaces (0.10.9-0.2+b3) ...
Setting up libutempter0:amd64 (1.2.1-2) ...
Setting up keyutils (1.6.1-2) ...
Setting up python3-pyasn1 (0.4.8-1) ...
Setting up python3-constantly (15.1.0-2) ...
Setting up xterm (366-1) ...
Setting up libnfsidmap2:amd64 (0.25-6) ...
Setting up libfaketime:amd64 (0.9.8-9) ...
Setting up ethtool (1:5.9-1) ...
Setting up faketime (0.9.8-9) ...
Setting up tcl8.6 (8.6.11+dfsg-1) ...
Setting up libtk8.6:amd64 (8.6.11-2) ...
Setting up screen (4.8.0-6) ...
Setting up nfs-common (1:1.3.4-6) ...

Creating config file /etc/idmapd.conf with new version
Adding system user `statd' (UID 117) ...
Adding new user `statd' (UID 117) with group `nogroup' ...
Not creating home directory `/var/lib/nfs'.
Created symlink /etc/systemd/system/multi-user.target.wants/nfs-client.target → 
/lib/systemd/system/nfs-client.target.
Created symlink /etc/systemd/system/remote-fs.target.wants/nfs-client.target → /
lib/systemd/system/nfs-client.target.
nfs-utils.service is a disabled or a static unit, not starting it.
Setting up python3-pyasn1-modules (0.2.1-1) ...
Setting up python3-service-identity (18.1.0-6) ...
Setting up epoptes-client (21.02-1) ...
Created symlink /etc/systemd/system/multi-user.target.wants/epoptes-client.servi
ce → /lib/systemd/system/epoptes-client.service.
Setting up tcl (8.6.11+1) ...
Setting up tk8.6 (8.6.11-2) ...
Setting up nfs-kernel-server (1:1.3.4-6) ...
Created symlink /etc/systemd/system/multi-user.target.wants/nfs-server.service →
 /lib/systemd/system/nfs-server.service.
Job for nfs-server.service canceled.

Creating config file /etc/exports with new version

Creating config file /etc/default/nfs-kernel-server with new version
Setting up python3-twisted (20.3.0-7) ...
Setting up tk (8.6.11+1) ...
Setting up x11vnc (0.9.16-7) ...
Setting up epoptes (21.02-1) ...
Adding group `epoptes' (GID 124) ...
Done.
Generating a RSA private key
......................++++
................................................................................
..............................................++++
writing new private key to '/etc/epoptes/server.key'
-----

A new OpenSSL certificate has been generated for epoptes.
Please ensure that you transfer /etc/epoptes/server.crt
to your clients by issuing `epoptes-client -c` from your
regular workstations or from your LTSP chroots.
Created symlink /etc/systemd/system/multi-user.target.wants/epoptes.service → /l
ib/systemd/system/epoptes.service.
Processing triggers for desktop-file-utils (0.26-1) ...
Processing triggers for initramfs-tools (0.140) ...
update-initramfs: Generating /boot/initrd.img-5.10.0-9-amd64
W: Possible missing firmware /lib/firmware/i915/skl_huc_2.0.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/skl_guc_33.0.0.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/bxt_huc_2.0.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/bxt_guc_33.0.0.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/kbl_huc_4.0.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/kbl_guc_33.0.0.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/glk_huc_4.0.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/glk_guc_33.0.0.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/kbl_huc_4.0.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/kbl_guc_33.0.0.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/cml_huc_4.0.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/cml_guc_33.0.0.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/icl_huc_9.0.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/icl_guc_33.0.0.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/ehl_huc_9.0.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/ehl_guc_33.0.4.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/tgl_huc_7.5.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/tgl_guc_35.2.0.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/tgl_huc_7.5.0.bin for module i91
5
W: Possible missing firmware /lib/firmware/i915/tgl_guc_35.2.0.bin for module i9
15
W: Possible missing firmware /lib/firmware/i915/bxt_dmc_ver1_07.bin for module i
915
W: Possible missing firmware /lib/firmware/i915/skl_dmc_ver1_27.bin for module i
915
W: Possible missing firmware /lib/firmware/i915/kbl_dmc_ver1_04.bin for module i
915
W: Possible missing firmware /lib/firmware/i915/glk_dmc_ver1_04.bin for module i
915
W: Possible missing firmware /lib/firmware/i915/cnl_dmc_ver1_07.bin for module i
915
W: Possible missing firmware /lib/firmware/i915/icl_dmc_ver1_09.bin for module i
915
W: Possible missing firmware /lib/firmware/i915/tgl_dmc_ver2_08.bin for module i
915
W: Possible missing firmware /lib/firmware/i915/rkl_dmc_ver2_02.bin for module i
915
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for libc-bin (2.31-13+deb11u2) ...
Processing triggers for man-db (2.9.4-2) ...
Processing triggers for mailcap (3.69) ...
Processing triggers for python3-twisted (20.3.0-7) ...
gpasswd: user 'administrator' does not exist
root@littledebi:/home/ee#

**Comment by alkisg** [comment]
Note: you can edit your message and wrap the output like this:

````
```
output goes here
```
````

It will display better that way.

OK, so, LTSP was installed successfully, I see no problems in the output. You may continue with the next steps of the https://ltsp.org/docs/installation/ page.

**Reply by knightflyer** [reply]
The missing firmware is not an issue?
I am re-installing debian.  Should I use the debian repository as is, or do the wget from the ltsp.org page?

**Comment by alkisg** [comment]
As I said above, the missing firmware messages aren't related to LTSP, you may contact Debian about them.
Debian CDs exist with "non-free firmware" included that may help, or you may install the firmware afterwards.

The issue is not related to LTSP at all, and it doesn't affect its functionality.

**Reply by knightflyer** [reply]
I understand, but after I run the wget, ltsp is not found (the executable)

**Comment by alkisg** [comment]
I did not see any error message with you running ltsp in the output, and failing... that's what I was asking for, the actual error message!

Anyway I think I understood what you mean. The ltsp binary is in /usr/sbin, and you're running `su` to become root instead of `su -`, so you don't have /usr/sbin in your PATH.

I.e. use `su -`, not `su`. This is documented in Debian (again, not related to LTSP at all!):
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=918754
/usr/share/doc/util-linux/NEWS.Debian.gz

**Reply by knightflyer** [reply]
Thanks for you patience. I admit my lack of fluency between the sudos and non sudoers between operating systems and their maintainers.

my apologies and gratitude for your time.

**Comment by knightflyer** [comment]
Okay I got a client to start to boot.
See img
I suppose I have to install some free radeon driver on the server?

**Comment by alkisg** [comment]
Try to go online to github discussions and reply from there. Now you replied by email, and the image wasn't attached.

**Reply by knightflyer** [reply]
![PXL_20211203_183658384](https://user-images.githubusercontent.com/84279099/144660567-e7c6c7a5-7432-48a4-92c1-cd26e7275cb6.jpg)
Did you get the image?

**Reply by knightflyer** [reply]
that firmare matches the earlier message.

**Comment by alkisg** [comment]
This again is not ltsp related, try to contact debian about it...

**Comment by knightflyer** [comment]
Will do.
